### ISSUE TITLE
Command bar menus with submenus don't announce that they are submenus

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1599]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

1. Search box doesn't say it's label "Search"
2. Buttons don't announce themselves as buttons.
3. Menu buttons should be "Button, has popups". See @cschlechty on how to do this correctly.

**Comment by betrue-final-final** [comment]
#1 does not repro in Narrator, but items with submenus don't announce that they are submenus

**Comment by betrue-final-final** [comment]
The submenu issue still repros.
